src/api_server/src/parsed_request.rs

```diff
@@ -115,6 +114,11 @@ impl ParsedRequest {
             }
         }
     }
+
+    /// Helper function to avoid boiler-plate code.
+    pub fn sync(vmm_action: VmmAction) -> ParsedRequest {
```

Comment:
Nit: I'm not sure about this name. Looks like an action. How about `from_vmm_action` or `create_sync` / `new_sync` ?

Reply:
done